tedana/metrics/collect.py

```diff
@@ -0,0 +1,140 @@
+"""
+Collect metrics.
+"""
+import logging
+
+import numpy as np
+import pandas as pd
+
+from .dependence import *
+from ._utils import determine_signs, flip_components, sort_df, apply_sort
+
+
+LGR = logging.getLogger(__name__)
+RepLGR = logging.getLogger('REPORT')
+RefLGR = logging.getLogger('REFERENCES')
+
+
+def generate_metrics(comptable, data_cat, data_optcom, mixing, mask, tes, ref_img, mixing_z=None,
+                     metrics=['kappa', 'rho'], sort_by='kappa', ascending=False):
```

Comment:
I think it's generally best to not have a mutable object as a default parameter. Can we set `metrics` to `None` and then do a check in the function?